Switch/Switch.m

```diff
@@ -81,8 +80,8 @@ -(void)setVisibleWidth:(CGFloat)visibleWidth
     _visibleWidth = visibleWidth;
     
     self.frame = CGRectMake(_origin.x, _origin.y, _visibleWidth, self.frame.size.height);
-    imgVw.frame = CGRectMake((_on ? 0 : -(_image.size.width/Scale - _visibleWidth)), 0,
-                             _image.size.width/Scale, _image.size.height/Scale);
+    imgVw.frame = CGRectMake((_on ? 0 : -(_image.size.width - _visibleWidth)), 0,
```

Comment:
To make this a little more readable try 
float x = _on ? 0 : -(_image.size.width - _visibleWidth) 
instead of nesting it in the CGRectMake